src/index.test.js

```diff
@@ -291,15 +348,19 @@ describe("Lumigo plugin (python)", () => {
 			});
 		});
 
-		test("it should wrap all functions after package initialize", async () => {
-			fs.pathExistsSync.mockReturnValue(true);
-			fs.readFile.mockReturnValue(`
+		describe("given the requirement.txt file exists", () => {
```

Comment:
What's the purpose of the test?

Reply:
it's not a separate test, just there so I can add the `fs` mock setup in a `beforeEach` so I have more predictability when it comes to when stubbed behaviours are added and removed. I find that when they're configured as part of the test body it doesn't always get cleared after the test for some reason.